scripts/bind/pic/template.hbs

```diff
@@ -41,14 +41,8 @@ impl PicCanisterTrait for {{PascalCase canister_name}}Pic {
 
 impl {{PascalCase canister_name}}Pic {
   {{#each methods}}
```

Comment:
We can simplify this branching code and eliminate the `unimplemented!()` now that we support arbitrary tuples.